trymito.io/pages/mito_pro_roadmap.tsx

```diff
@@ -0,0 +1,158 @@
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import Image from 'next/image'
+import Footer from '../components/Footer/Footer';
+import Header from '../components/Header/Header';
+import securityStyles from '../styles/Security.module.css'
+import pageStyles from '../styles/Page.module.css'
+import titleStyles from '../styles/Title.module.css'
+import GithubButton from '../components/GithubButton/GithubButton';
+import DownloadCTACard from '../components/CTACards/DownloadCTACard';
+
+import FlagIcon from '../public/icon-squares/FlagIcon.svg'
+
+
+const Mito_Pro_Roadmap: NextPage = () => {
+
+  return (
+    <>
+      <Head>
+        <title>Mito | Roadmap </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
+      
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+
+            <section className={titleStyles.title_card + ' ' + titleStyles.grid_card}>
+                <h1 className={titleStyles.title}>
+                    Mito Pro Roadmap 
+                </h1>
+                <p className={titleStyles.description}>
+                    We&apos;re constantly releasing new features to speed up your data analysis
```

Comment:
I think this page should also list the pro features that we already have deployed. Perhaps they don't need to be in this format, but might make sense in a list ala the plans page? 

I think that having _all_ the information about the pro features here makes a ton of sense, as users aren't gonna think "Oh what are the new things too" - they are just gonna wonder "what's the deal with Pro" and want to go to one place. 

On that front, might we want to move some of the Pro FAQ here? 

I think that the things I would include on this page are:
1. High-level information about Pro and Enterprise at the top. 
2. A list of features we already have. 
3. A list of coming features (what this page currently is). 
4. FAQ on the Pro and Enterprise plans.

Let me know what you think of this!

Reply:
As discussed, instead of duplicating info on this new page, we're just going to move the roadmap to the plans page